apps/hash-external-services/docker-compose.yml

```diff
@@ -163,11 +163,6 @@ services:
       RUST_BACKTRACE: 1
     ports:
       - "${HASH_GRAPH_API_PORT}:4000"
-    healthcheck:
-      test: ["CMD", "/hash-graph", "server", "--healthcheck"]
-      interval: 2s
-      timeout: 2s
-      retries: 10
```

Comment:
For anyone wondering, this was moved to the dockerfile of the graph

Reply:
I'm sorry, I know that you specifically asked about this change and I approved this change but I just noticed that this has to be reverted. The healthcheck here is specifically for the `server` subcommand, not a general purpose healthcheck. Not all subcommands has a healthcheck. I'll open a PR for that.

Reply:
Good point, cheers